File: 40-50 arası örnekler/49.bazal_metabolizma_hizi.py
k=int(input("Kilonuzu giriniz:  "))
b=int(input("Boyunuzu giriniz: "))
y=int(input("Yaşınızı giriniz: "))
c=str(input("Cinsiyetinizi giriniz: "))

if (c=="E"):
    bmh=66.473+13.7516*k+5.0033*b-6.775*y
    print("bmh=",bmh)
else:
    bmh=655.0955+9.5634*k+1.8496*b-4.6756*y
    print("bmh=",bmh)

File: c/40-50 arası örnekler/49_bazal_metabolizma_hizi.c
#include <stdio.h>

int main()
{
    int k, b, y, bmh;
    char c;
    printf("Kilonuzu giriniz: ");
    scanf("%d", &k);
    printf("Boyunuzu giriniz: ");
    scanf("%d", &b);
    printf("Yasinizi giriniz: ");
    scanf("%d", &y);
    printf("Cinsiyetizi giriniz: ");
    scanf("%s", &c);

    if (c == 'E' || c == 'e')
    {
        bmh = (66.473 + 13.7516 * k + 5.0033 * b - 6.775 * y);
    }
    else
        bmh = (665.0955 + 9.5634 * k + 1.8496 * b - 4.6756 * y);
    printf("Bazal metabolizma hizi: %d\n", bmh);

    return 0;
}